src/gitlab.ts

```diff
@@ -37,12 +37,12 @@ export const runCommandInGitlabPipeline = async (ctx: Context, task: Task) => {
         ...task.gitlab.job,
         script: [
           // prettier-ignore
-          'if [ "${PIPELINE_SCRIPTS_REPOSITORY:-}" ]; then' +
-            'if [ "${PIPELINE_SCRIPTS_REF:-}" ]; then' +
-              getPipelineScriptsCloneCommand({ withRef: true }) + ';' +
```

Comment:
I still don't get why do you complicate this with a whole 8-line function, which allows you to save 19 characters (`git clone --depth 1`), while wasting 49 chars on calling it. 

The whole script is static, it doesn't change based on environment. Just inline everything, it'll be more compact and easier to read.

Reply:
You don't seem to be noticing, from this comment and https://github.com/paritytech/try-runtime-bot/pull/57#discussion_r867893084, that it's not a single argument to be duplicated, it's many. `git clone --depth 1 "$PIPELINE_SCRIPTS_REPOSITORY" "$PIPELINE_SCRIPTS_DIR"` has 5 arguments in total. I'm not willing to duplicate that in two places.

Reply:
Ah, I got the ternary wrong.
Still, 252 char function which saves 54 chars, with a waste of 49 on the call itself, but okay